src/index.js

```diff
@@ -8,11 +8,20 @@ const terser = require("terser");
 const tsconfigPaths = require("tsconfig-paths");
 const { loadTsconfig } = require("tsconfig-paths/lib/tsconfig-loader");
 const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
+const nodeLibsBrowser = require('node-libs-browser');
```

Comment:
I think I misunderstood the problem earlier 😅 

This package's README says "The node core libs for in-browser usage."

We don't need ncc output to work in the browser so I think we can remove this 👍